internal/tools/aws/cluster_management.go

```diff
@@ -249,7 +249,6 @@ func (a *Client) releaseVpc(clusterID string, logger log.FieldLogger) error {
 	}
 
 	for _, subnet := range publicSubnets {
-		logger.Debugf("Untagging subnet %s", *subnet.SubnetId)
```

Comment:
Just curious if you meant to remove this line?

Reply:
Yep! Redundant logs are redundant. ;)

Reply:
Perfect, I figured it was something like that